llama_index/data_structs/data_structs.py

```diff
@@ -163,9 +163,9 @@ def get_type(cls) -> IndexStructType:
 class IndexDict(IndexStruct):
     """A simple dictionary of documents."""
 
-    # mapping from vector store id to node id
+    # mapping from vector store id to node doc_id
     nodes_dict: Dict[str, str] = field(default_factory=dict)
-    # mapping from doc_id to vector store id
+    # mapping from node doc_id to vector store id
     doc_id_dict: Dict[str, List[str]] = field(default_factory=dict)
 
     # TODO: temporary hack to store embeddings for simple vector index
```

Comment:
curious: do we still need the `embeddings_dict`? Might as well clear it out in this PR if not.

Reply:
removing this (and embeddings dict) may cause some compatibility issues.... I'll leave it for now, I'm not sure we want to break everyone's vector index right now

Reply:
Sure, let's just mark it being deprecated with comments then.

And I think we can remove the logic related to `doc_id_dict` in `add_node` and `delete_node`? since it would not break existing vector indexes.